Verbali

RakuJa opened the pull request.

RakuJa committed:
Aggiunto verbale 09 11 2020

RakuJa committed:
Caricamento verbali 09,16,23 - 11- 2020

Edivad99 commented on Interni/Verbali Interni/2020_11_09/components/verbale.tex:
```diff
@@ -0,0 +1,97 @@
+
+\begin{itemize}
+
+	\item Presentazione membri del gruppo
```
Sugli elenchi puntati manca il ; e . alla fine

RakuJa replied:
Fixed.

RakuJa committed:
Small code refactoring

Aggiunti DataMeeting e DataDoc ed implementati, utilizzo maggiore anche dei comandi Gruppo e Mail.

RakuJa committed:
Aggiunta punteggiatura ad elenchi & grassetto ordine giorno

Edivad99 commented on Interni/Verbali Interni/2020_11_09/components/verbale.tex:
```diff
@@ -0,0 +1,96 @@
+
+\subsection{Votazione sulla registrazione delle chiamate}
+
+Sotto proposta di Daniele è stato deciso all’unanimità di registrare le chiamate, questo perchè la registrazione ed il salvataggio delle chiamate risultano un peso trascurabile mentre i benifici che esse possono portare sono diversi.
```
Scritto male benefici

Edivad99 replied:
Il simbolo dell'apostrofo a Latex non gli va bene🤷‍♂️

Edivad99 replied:
Sottocanali non va attaccato, staccato o con la linea in mezzo

RakuJa replied:
Fixato 1 e 3, per il 2 ho cambiato il simbolo dell'apostrofo, spero non lo riconverta automaticamente

Edivad99 commented on Interni/Verbali Interni/2020_11_09/informazioni.tex:
```diff
@@ -0,0 +1,25 @@
+
+\newcommand{\Titolo}{Verbale interno \DataMeeting{}}
+
+\newcommand{\Gruppo}{MercurySeven}
```
Spazio

RakuJa replied:
Fixed

Edivad99 commented on Interni/Verbali Interni/2020_11_16/informazioni.tex:
```diff
@@ -0,0 +1,25 @@
+
+\newcommand{\Titolo}{Verbale interno \DataMeeting{}}
+
+\newcommand{\Gruppo}{MercurySeven}
```
Spazio

RakuJa replied:
Fixed

Edivad99 commented on Interni/Verbali Interni/2020_11_23/informazioni.tex:
```diff
@@ -0,0 +1,25 @@
+
+\newcommand{\Titolo}{Verbale interno \DataMeeting{}}
+
+\newcommand{\Gruppo}{MercurySeven}
```
Spazio

RakuJa replied:
Fixed

Edivad99 commented on Interni/Verbali Interni/2020_11_23/components/verbale.tex:
```diff
@@ -0,0 +1,114 @@
+
+È stato creato un template per i documenti che verranno generati con \LaTeX. È stata inoltre fornita ai membri con lacune nell'uso del linguaggio una breve spiegazione delle basi di \LaTeX.
+
+\subsection{Impostazione repository github}
```
GitHub

RakuJa replied:
Fixed

Edivad99 commented:
Manca il punto in molti paragrafi

Edivad99 commented on Interni/Verbali Interni/2020_11_23/components/verbale.tex:
```diff
@@ -0,0 +1,114 @@
+
+\section{Resoconto}
+
+\subsection{Discussione e Realizzazione template LaTeX}
```
Scrivere \LaTeX

RakuJa replied:
Fixed

Edivad99 commented on Interni/Verbali Interni/2020_11_23/components/verbale.tex:
```diff
@@ -0,0 +1,114 @@
+
+\begin{itemize}
+
+	\item \textbf{2021-01-11:} Consegna documentazione
```
Manca ; e .

RakuJa replied:
Fixed

RakuJa committed:
Fix vari

Edivad99 commented on Interni/Verbali Interni/2020_11_09/components/verbale.tex:
```diff
@@ -0,0 +1,96 @@
+\section{Infomazioni generali}
```
Sbagliato a scrivere Informazioni

RakuJa replied:
Fixed

Edivad99 commented on Interni/Verbali Interni/2020_11_23/components/verbale.tex:
```diff
@@ -0,0 +1,114 @@
+\section{Infomazioni generali}
```
Sbagliato a scrivere Informazioni

RakuJa replied:
Fixed

Edivad99 commented on Interni/Verbali Interni/2020_11_16/components/verbale.tex:
```diff
@@ -0,0 +1,97 @@
+\section{Infomazioni generali}
```
Sbagliato a scrivere Informazioni

RakuJa replied:
Fixed

Edivad99 commented on Interni/Verbali Interni/2020_11_09/components/verbale.tex:
```diff
@@ -0,0 +1,96 @@
+
+\subsection{Impostazione dei canali di comunicazione}
+
+È stato creato un canale Slack ed è stato opportunamente impostato in vari sotto canali per permettere una migliore organizzazione ed una più chiara comunicazione tra membri
```
Manca il punto

RakuJa replied:
Fixed

Edivad99 commented on Interni/Verbali Interni/2020_11_16/components/verbale.tex:
```diff
@@ -0,0 +1,97 @@
+
+\subsection{Logo del gruppo}
+
+È stato approvato il logo del gruppo disegnato da Lucrezia, esso presenta due versioni possibili versioni, una che presenta il testo "\Gruppo{}" adatta alle intestazioni dei documenti mentre l'altra senza nessun testo usata per le miniature
```
Manca il punto

RakuJa replied:
Fixed

Edivad99 commented on Interni/Verbali Interni/2020_11_16/components/verbale.tex:
```diff
@@ -0,0 +1,97 @@
+
+\subsection{Votazione su strumenti di versionamento e per stesura documenti}
+
+È stato deciso di utilizzare Git come sistema di versionamento del progetto e GitHub come piattaforma di versionamento. Per la stesura dei documenti vi era molte alternative ma si è deciso di optare per l'utilizzo di \LaTeX
```
Manca il punto

RakuJa replied:
Fixed

Edivad99 commented on Interni/Verbali Interni/2020_11_23/components/verbale.tex:
```diff
@@ -0,0 +1,114 @@
+
+Si è deciso di approcciare lo Studio di Fattibilità con la seguente modalità: 
+\begin{itemize}
+	\item Spartizione della esposizione dei capitolati tra i singoli membri	;
```
Spazio tra membri e ;

RakuJa replied:
Fixed

RakuJa committed:
Errore grammaticale "informazioni"

RakuJa committed:
Fix missing .

RakuJa committed:
Fix minori

RakuJa committed:
Punto mancante

RakuJa committed:
Update registro modifiche

Edivad99 commented on Interni/Verbali Interni/2020_11_16/components/verbale.tex:
```diff
@@ -0,0 +1,97 @@
+
+\subsection{Nome del gruppo}
+
+Dopo una attenta riflessione individuale nella quale ogni membro ha pensato a dei nomi per il gruppo, essi sono stati discussi insieme e tra i molti nomi proposti,il gruppo ha scelto all’unanimità “\textit{\Gruppo{}}”.
```
Solito problema dell'apostrofo all’unanimità

RakuJa committed:
Fixed dates (wrong month)
